pkgs/servers/monitoring/grafana/plugins/grafana-oncall-app/default.nix

```
@@ -0,0 +1,15 @@
+{ grafanaPlugin, lib }:
+
+grafanaPlugin rec {
+  pname = "grafana-oncall-app";
+  version = "1.5.0";
+  zipHash = {
+    x86_64-linux = "sha256-DOQ3W77fhLedgXH1VHW/JTjnDi0PxeASdsSAsnyG2xg=";
+  };
```

Your previous version had hashes for all 4 architectures, not just `x86_64-linux`.

But given this plugin does not contain precompiled (architecture-specific) binaries and

https://grafana.com/api/plugins/grafana-oncall-app/versions/1.5.0/download?os=linux&arch=amd64
and
https://grafana.com/api/plugins/grafana-oncall-app/versions/1.5.0/download

return exactly the same ZIP, file structure, contents and recursive NAR hash, this should probably use

```suggestion
  zipHash = "sha256-DOQ3W77fhLedgXH1VHW/JTjnDi0PxeASdsSAsnyG2xg=";
```
instead.